examples/throttling.rb

```diff
@@ -0,0 +1,68 @@
+# simple.rb
+# this example shows you a very basic message/handler setup
```

Comment:
I think this description should mention the throttling behaviour we want to test.

Reply:
fixed